app/home/homeController.js

```diff
@@ -68,13 +68,25 @@
 
         function loadEvents() {
             EventService.getEvents().then(function success(response) {
-                homeCtrl.events = response.data;
+                homeCtrl.events = activeEvents(response.data);
             }, function error(response) {
                 MessageService.showToast(response.data.msg);
                     $state.go('app.home');
             });
         }
 
+        function activeEvents(allEvents){
+            var now = new Date();
+            var actualEvents = [];
+            for (var i = 0; i < allEvents.length - 1; i++) {
```

Comment:
pq usar for com indice se vc está passando por todo o array e só usa o índice para pegar o elemento? Semanticamente fica melhor um foreach, raras a vezes que se precisa realmente de um for com índice, como por exemplo, precisa-se calcular alguma coisa com o índice corrente, ou pular de 2 em 2, ou executar uma varredura que não passa por todos os elementos.